Minor editorial, grammatical, and spelling improvments to docs (#1701)
vercel/turborepo: docs/pages/docs/guides/workspaces.mdx

Workspaces are a feature implemented by package managers to assist with working on multiple apps and packages in the same codebase.

Turborepo is compatible with three package managers ([npm](https://docs.npmjs.com/cli/v8/using-npm/workspaces#description), [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/workspaces/), and [pnpm](https://pnpm.io/workspaces)), each with its own implementation of workspaces and functionality. Your package manager will determine how your apps and packages are organized in your workspaces and run in your turborepo.

## Configuring workspaces

> To use workspaces, you'll first need to declare their file system locations to your package manager.

Reviewer: ```suggestion
To use workspaces, you must first declare their file system locations to your package manager.
```

> A common convention we recommend is having top-level `apps/` and `packages/` directories. This isn't a requirement but just a suggested directory structure. The `apps` folder can contain workspaces for launchable apps, such as a Next.js or Svelte app. The `packages` folder can contain workspaces for packages used by either an app or another package.

Reviewer 2: ```suggestion
A common convention we recommend is having top-level `apps/` and `packages/` directories. This isn't a requirement but just a suggested directory structure. The `apps` folder can contain workspaces for launchable apps, such as a [Next.js](https://nextjs.org/) or [Svelte](https://svelte.dev/) app. The `packages` folder can contain workspaces for packages used by either an app or another package.
```

> A common convention that we recommend is to have top-level `apps/` and `packages/` directories. The `apps` folder can contain workspaces for launchable apps, such as a Next.js or Svelte app. The `packages` folder can contain workspaces for packages that are used by either an app, or another package. This isn't a requirement, but just a suggested directory structure.

Reviewer: ```suggestion
A common convention we recommend is having top-level `apps/` and `packages/` directories. This isn't a requirement but just a suggested directory structure. The `apps` folder can contain workspaces for launchable apps, such as a Next.js or Svelte app. The `packages` folder can contain workspaces for packages used by either an app or another package.
```

├─ sdk/
```

In the example above, all directories inside `monorepo/apps/` and `monorepo/packages/` are workspaces, and the `monorepo/docs` directory itself is also a workspace. `monorepo/sdk/` is _not_ a workspace, as it is not included in the workspace configuration.

## Managing workspaces

> When you move, delete, or rename your workspaces, you will have to make sure that all folders linked within your `package.json` matches. Anytime you change the configuration of your workspace, make sure all the dependencies of the workspace are also updated. Re-run your package manager's install command to check your configuration. If there are any problems after that, you may have to delete your `node_modules` folder and run an install again.

Reviewer: ```suggestion
When you move, delete, or rename your workspaces, you must ensure that all folders linked within your `package.json` should match. Anytime you change your workspace configuration, provide all the dependencies of the workspace that are updated. Re-run your package manager's install command to check your configuration. If there are problems after that, you may have to delete your `node_modules` folder and re-run an install.
```

## Managing dependencies

> To manage dependencies within workspaces from the root of your monorepo, run commands that only manage the dependencies of each workspace rather than for the entire monorepo.

Reviewer: ```suggestion
To manage dependencies within workspaces from the root of your monorepo, run commands that only manage each workspace's dependencies rather than the entire monorepo.
```
